seed/data_importer/tests/integration/test_footprints_import.py

```diff
@@ -177,8 +177,11 @@ def test_demo_v2(self):
         self.assertEqual(tdq.results[tax_lot_3.id]['data_quality_results'][0]['detailed_message'], "'' is not a valid geometry")
 
         pdq = DataQualityCheck.retrieve(self.org.id)
-        pdq.check_data('PropertyState', [property_1, property_2, property_3])
-        self.assertEqual(pdq.results.get(property_1.id, None), None)
+        pdq.check_data('PropertyState', [property_1 , property_2, property_3])
+        # For property_1, no error message of null 'Conditioned Floor Area' reported if no 'not_null' condition is set
+        for rule in pdq.rules.filter(table_name='PropertyState', field='conditioned_floor_area'):
+            self.assertNotEqual(rule.condition, 'not_null')
```

Comment:
I'm thinking this test is unrelated to the footprint import. Is that right? If so, I'm fine to keep it here but worth noting it is very confusing to see it here.

Reply:
I will remove it and move it to data_quality_tests